p3/data/_validation.py

```diff
@@ -1,49 +1,48 @@
-# Copyright (c) 2022-2023 Intel Corporation
-# SPDX-License-Identifier: MIT
```

Comment:
Even though you have modified these files, the copyright notice needs to stay in place as per the license terms.